package.json

```diff
@@ -8,13 +8,14 @@
     "eslint": "^7.32.0",
     "eslint-config-airbnb-base": "^14.2.1",
     "eslint-plugin-import": "^2.24.2",
-    "hint": "^6.1.4",
-    "stylelint": "^13.13.1",
-    "stylelint-config-standard": "^21.0.0",
+    "hint": "^6.1.4"
+  },
+  "devDependencies": {
+    "stylelint": "13.3.x",
+    "stylelint-config-standard": "20.0.x",
     "stylelint-csstree-validator": "^1.9.0",
-    "stylelint-scss": "^3.20.1"
+    "stylelint-scss": "3.17.x"
   },
-  "devDependencies": {},
   "scripts": {
```

Comment:
**Kindly work on these changes:**
- Your package.json file it's missing scripts to run the server and build your project.